src/models/liturgical-day/liturgical-day.types.ts

```diff
@@ -195,7 +195,7 @@ export type LiturgicalDayMetadata = {
 };
 
 /**
- * The source of the liturgical day.
```

Comment:
L198-206 seems to duplication of L125-133. We might want to be more specific in order to better document both parts of the code.

That said it is not a major issue. wink:

Reply:
I kept the documentation on the object property (useful when editing a calendar definition, because smart IDEs adds text hint on the currently edited property).

I've reduced the documentation on the object type itself.